src/js/lib/crypto.js

```diff
@@ -0,0 +1,16 @@
+import wallet from 'ethereumjs-wallet'
+import wif from 'wif'
+
+// @summary - Decodes WIF encoded key and returns the key and address
```

Comment:
Perhaps we could use comment block here?

Reply:
Changed, thank you.